common-test-lib/src/main/java/com/google/gct/idea/testing/TestUtils.java

```diff
@@ -127,6 +129,7 @@ public static MockProject mockProject(@Nullable PicoContainer container) {
     return new MockProject(container, getParentDisposableForCleanup());
   }
 
+  @SuppressFBWarnings(value="DM_STRING_CTOR", justification="Warning due to string concatenation")
```

Comment:
AFAK, FindBugs sees actual bytecode instead of source, so I guess it's explicit that things like new String("str") are happening when "+" are compiled into bytecode. We could check the bytecode directly using a debugger if this is really happening, but I don't know how to do so. Perhaps with some compiler optimization options, this could be avoided?